packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -324,3 +347,33 @@ export interface IContextualMenuItem {
    */
   [propertyName: string]: any;
 }
+
+export interface IContextualMenuSection extends React.Props<ContextualMenu> {
+
+  /**
+   * The items to include inside the section.
+   */
+  items: IContextualMenuItem[];
```

Comment:
That's a good point. Talking about it with our PM (WAC), it seems like we don't want to support that.
The problem is that I don't really know how to prevent it.